Composer/packages/extensions/visual-designer/src/utils/cursorTracker/tabMove.ts

```diff
@@ -32,20 +33,38 @@ function findSelectableParent(element: SelectorElement, elementList: SelectorEle
 
 export function handleTabMove(currentElement: SelectorElement, selectableElements: SelectorElement[], command: string) {
   let nextElement: SelectorElement;
+  const selectableChildren = findSelectableChildren(currentElement, selectableElements);
+  const selectableParent = findSelectableParent(currentElement, selectableElements);
   if (command === KeyboardCommandTypes.Cursor.MoveNext) {
-    const selectableChildren = findSelectableChildren(currentElement, selectableElements);
     if (selectableChildren.length > 0) {
       // Tab to inner selectable element.
       nextElement = selectableChildren[0];
     } else {
-      // Perform like presssing down arrow key.
-      nextElement = locateNearestElement(currentElement, selectableElements, Direction.Down, ['isNode', 'isEdgeMenu']);
+      const hasInlineLinkElement =
+        currentElement.selectedId.endsWith('dot') &&
+        selectableElements.find(element => element.selectedId === selectableParent?.selectedId + 'link');
```

Comment:
There's probably a better way to write this without duplicating this expression (on line 48 and also on line 63, although 'link' there is placed with 'dot'). I won't block approval on needing it fixed, but if you could clean it up, it'd be nice.

Reply:
Refactor.